lighthouse-docker/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "lighthouse_docker",
+  "description": "Lighthouse in Docker",
+  "author": "Eric Bidelman <ebidel@>",
+  "license": "Apache-2.0",
+  "main": "server.js",
+  "scripts": {
+    "start": "node server.js",
+    "restart": "npm run build && npm run serve",
+    "build": "./docker_build.sh",
+    "build:headful": "./docker_build.sh --headful",
+    "serve": "./docker_run.sh",
+    "serve:headful": "./docker_run.sh --headful",
+    "chrome:version": "docker run -it --rm --cap-add=SYS_ADMIN lighthouse_docker https://example.com --fast --quiet --output=json | node -e \"let f = ''; process.stdin.on('data', d => f += d); process.stdin.on('close', () => console.log(JSON.parse(f).userAgent));\""
+  },
+  "dependencies": {
+    "body-parser": "^1.18.1",
```

Comment:
should this be two different `package.json` files? It ends up kind of confusing that this is governs outside the container and inside, but it looks like the dependencies are only internally used while the npm scripts are only externally used, so maybe it could just be split? I don't know what the usual docker approach is for this